forum/startups/helpers.py

```diff
@@ -0,0 +1,78 @@
+from django.db.models import Q
+from .models import Startup
+from .serializers import StartupSerializer, StartupSizeSerializer
+
+from projects.models import Project, Industry
+from projects.serializers import ProjectSerializer, IndustrySerializer
+
+
+def select_startups_by_search_string(search_string):
+    return Startup.objects.filter(
+        Q(name__icontains = search_string) |
+        Q(location__icontains = search_string) |
+        Q(description__icontains = search_string)
+    )
+
+
+def filter_startups(query_params):
+    industry_name = query_params.get("industry")
+    max_budget = query_params.get("max-budget")
+    size = query_params.get("size")
+    
+    startups = Startup.objects.filter()
+
+    if industry_name:
+        industry = Industry.objects.filter(name=industry_name).first()
+        projects = Project.objects.filter(industries=industry)
+        startups_ids = [proj.startup.startup_id for proj in projects]
```

Comment:
You may use `values_list` for more efficient querying of related startup IDs:
```
startup_ids = projects.values_list('startup_id', flat=True)
```

Reply:
Updated getting startup_ids.